loopd/utils.go

```diff
@@ -39,9 +40,10 @@ func getClient(config *Config, lnd *lndclient.LndServices) (*loop.Client,
 
 func getLiquidityManager(client *loop.Client) *liquidity.Manager {
 	mngrCfg := &liquidity.Config{
-		AutoloopTicker: ticker.NewForce(liquidity.DefaultAutoloopTicker),
-		LoopOut:        client.LoopOut,
-		LoopIn:         client.LoopIn,
+		AutoloopTicker:                 ticker.NewForce(liquidity.DefaultAutoloopTicker),
+		AutoloopBudgetRefreshTimestamp: time.Now(),
```

Comment:
Starting with `Now()` would mean that we need to wait a whole refresh period right?

Reply:
You don't need to wait for it to perform loops, it means that a new budget-cycle has just started.